Executive.js

```diff
@@ -29,15 +29,14 @@ class Executive {
 	**/
 	initGame() {
 		document.getElementById("firstPlayer").value = document.getElementById("Player1").value;
+		document.getElementById("firstPlayer").disabled = true;
```

Comment:
These don't need to be disabled. If we don't want the user to be able to edit their name during gameplay, we could just not use text inputs there.